src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -27,16 +37,89 @@ private void initMenu() {
 
         this.add(this.importMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-        // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
-
         this.add(this.settingsMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-
         this.add(this.helpMenu);
+
+        // Add function to import menu
+        importMenu.addMenuListener(new MenuAdaptor(){
```

Comment:
I don't think you are doing the right way of designing the menu.

I suggest you read https://docs.oracle.com/javase/tutorial/uiswing/components/menu.html first and then go back recode this part.

For example, on the menu bar, there should be a File menu. The import bill `MenuItem` should be in the File menu, instead of laying out directly in the menu bar.

Similarly, there should be a Setting menu in the menubar. And the "change color and font" `MenuItem`s should be in the Setting menu. BTW, it deserves a better name like "Appearance".